### ISSUE TITLE
Incorrect class specified in SetupIntentPaymentMethodOptionsOptions.Card property

### ISSUE THREAD

**Issue by BB1001** [stripe/stripe-dotnet#1735]
In file stripe-dotnet/src/Stripe.net/Services/SetupIntents/SetupIntentPaymentMethodOptionsOptions.cs 

A PaymentIntentPaymentMethodOptionsCardOptions type is specified for the Card property, it should be a SetupIntentPaymentMethodOptionsCardOptions type.

### Currently:
```
public class SetupIntentPaymentMethodOptionsOptions : INestedOptions
    {
        /// <summary>
        /// Configuration for any card payments attempted on this SetupIntent.
        /// </summary>
        [JsonProperty("card")]
        public PaymentIntentPaymentMethodOptionsCardOptions Card { get; set; }
    }
```

### Should be:
```
public class SetupIntentPaymentMethodOptionsOptions : INestedOptions
    {
        /// <summary>
        /// Configuration for any card payments attempted on this SetupIntent.
        /// </summary>
        [JsonProperty("card")]
        public SetupIntentPaymentMethodOptionsCardOptions Card { get; set; }
    }
```

**Comment by remi-stripe** [comment]
@BB1001 Thanks a lot for the report and sorry about the troubles here. We'll get this fixed in the next major version to avoid breaking anyone who made things work with the incorrect name.

**Comment by ob-stripe** [comment]
This is fixed in the latest major version, 30.0.0.
